src/main/resources/application.properties

```diff
@@ -19,7 +19,9 @@ idam.s2s-auth.totp_secret=${DATA_STORE_IDAM_KEY:AAAAAAAAAAAAAAAB}
 idam.s2s-auth.microservice=ccd_data
 idam.s2s-auth.url=${IDAM_S2S_URL:http://localhost:4502}
 
-migration.idam.username= ${IDAM_USE}
+migration.idam.username= ${IDAM_USER}
 migration.idam.password= ${IDAM_PASS}
 migration.caseType= ${MIGRATION_CASETYPE}
-migration.caseId= ${MIGRATION_CASETID:}
```

Comment:
@kiran-yenigala-hmcts You can remove the references of migration.caseId from Readme file

Reply:
removed